internal/tools/securitypolicy/main.go

```diff
@@ -78,9 +79,15 @@ func main() {
 	}
 }
 
+type EnvironmentVariableRule struct {
+	Strategy string `toml:"strategy"`
+	Rule     string `toml:"rule"`
```

Comment:
I'd make a new type for the consts mentioned in the other comment "EnvVarRule" or similar

Reply:
Will do for the 2nd half of September. An issue has been added.